### DISCUSSION TITLE
"Document" Content Type added by default but should not

### DISCUSSION THREAD

**Discussion by luisartg** [pnp/pnpframework#362] [Q&A]
I'm creating various document libraries in SharePoint. These libraries contain a defined set Content Types. However, when I extract the template of these libraries and then apply them to a new site, the new libraries are automatically adding a Content Type called "Document" which also adds itself to the New button. Why could this be happening?  The template should be copying the library as is, so why is adding a Content Type that is not in the original library or the template produced.

You can see the template I'm using: https://gist.github.com/Luis-Art-Guerra/a626257de4c646576c4c74e90d6255fb

**Comment by jansenbe** [comment]
The "Document" content type is the default content type for any document library and therefore it's added by default when the library is created. If you don't want that you need to explicitly remove it in your template or set the `RemoveExistingContentTypes` property to true.
